documentation-support/src/main/java/tech/picnic/errorprone/documentation/Json.java

```diff
@@ -0,0 +1,45 @@
+package tech.picnic.errorprone.documentation;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect.Visibility;
+import com.fasterxml.jackson.annotation.PropertyAccessor;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.guava.GuavaModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import com.google.errorprone.annotations.FormatMethod;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Path;
+
+/**
+ * Utility class that offers mutually consistent JSON serialization and deserialization operations,
+ * without further specifying the exact schema used.
+ */
+final class Json {
+  private static final ObjectMapper OBJECT_MAPPER =
+      new ObjectMapper()
+          .setVisibility(PropertyAccessor.FIELD, Visibility.ANY)
+          .registerModules(new GuavaModule(), new ParameterNamesModule());
```

Comment:
This logic is moved to a separate class, not only because the tests use it, but also because other yet-to-backported code relies on it.